# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fetch/metadata/window-open.https.sub.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/fetch/metadata/window-open.https.sub.html.ini
--- a/testing/web-platform/meta/fetch/metadata/window-open.https.sub.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/fetch/metadata/window-open.https.sub.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,15 +1,16 @@
 [window-open.https.sub.html]
   expected:
-    if not debug and (os == "linux") and not webrender and (processor == "x86"): ["ERROR", "OK"]
-    if not debug and (os == "win") and webrender and not fission: ["ERROR", "OK"]
+    if (os == "linux") and webrender and not debug: ["ERROR", "OK"]
     [OK, ERROR]
   [Cross-site window, forced, reloaded]
     expected:
-      if not webrender and (processor == "x86_64") and (os == "win") and debug: ["FAIL", "PASS"]
+      if (os == "win") and debug and not webrender and (processor == "x86_64"): ["FAIL", "PASS"]
+      if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["FAIL", "PASS"]
       [PASS, FAIL]
 
   [Same-site window, forced, reloaded]
     expected:
+      if (os == "android") and debug and not webrender: ["FAIL", "PASS"]
       if (os == "mac") and debug: PASS
       if os == "win": PASS
       [PASS, FAIL]